examples/nixnet/can-signal-single-point-output.py

```diff
@@ -0,0 +1,125 @@
+r""" Write Signal Data.
+
+ This example writes a signal value when a keyboard character is pressed.
+ This is used to demonstrate a signal single point output session. 
+ This example uses hardcoded signal names that use the NIXNET_example database. 
+
+The gRPC API is built from the C API. NI-XNET documentation is installed with the driver at:
+  C:\Users\Public\Documents\National Instruments\NI-XNET\Documentation\NI-XNET Manual.chm
+Getting Started:
+
+To run this example, install "NI-XNET Driver" on the server machine:
+  https://www.ni.com/en-in/support/downloads/drivers/download.ni-xnet.html
+
+For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC
+Client" wiki page:
+  https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+
+Refer to the NI XNET gRPC Wiki for the latest C Function Reference:
+  https://github.com/ni/grpc-device/wiki/NI-XNET-C-Function-Reference
+
+ Running from command line:
+Server machine's IP address and port number can be passed as separate command line
+arguments.
+  > python can-signal-single-point-output.py <server_address> <port_number>
+If they are not passed in as command line arguments, then by default the server address will be
+"localhost:31763"
+"""
+
+import sys
+
+import getch
+import grpc
+import nixnet_pb2 as nixnet_types
+import nixnet_pb2_grpc as grpc_nixnet
+
+SERVER_ADDRESS = "localhost"
+SERVER_PORT = "31763"
+
+# Read in cmd args
+if len(sys.argv) >= 2:
+    SERVER_ADDRESS = sys.argv[1]
+if len(sys.argv) >= 3:
+    SERVER_PORT = sys.argv[2]
+
+# Parameters
+INTERFACE = "CAN1"
```

Comment:
Consider; We usually allow the device name to be passed in as a command line argument as well. Does that work in this case? Or are there just too many names that need to match your MAX config?

Reply:
Yeah, we have implemented the same as passing the device name as interface in command line argument.